routes/tools_routes.js

```diff
@@ -69,47 +66,48 @@ async function topCards(filter) {
   }
 
   const oracleIds = [...oracleIdMap.keys()];
-  const query = filter ? { oracleId: { $in: oracleIds } } : {};
+  const query = filter
+    ? { $and: [{ oracleId: { $in: oracleIds } }, { 'current.picks': { $gt: MIN_PICKS } }] }
+    : { 'current.picks': { $gt: MIN_PICKS } };
   const selectedVersions = new Map(
     [...oracleIdMap.entries()].map(([oracleId, versions]) => [
       oracleId,
       carddb.getFirstReasonable(versions.map(({ _id }) => _id)),
     ]),
   );
 
-  const dataQ = Promise.all(
-    ['rating', 'elo', 'picks', 'cubes'].map(async (field) => {
-      const sorted = await CardHistory.find(query).sort(`-current.${field}`).limit(MAX_RESULTS).lean();
-      return sorted
-        .filter(({ oracleId }) => selectedVersions.has(oracleId))
-        .map(({ oracleId, current }) => {
-          const { rating, elo, picks, cubes } = current;
-          const qualifies = Number.isFinite(picks) && picks > MIN_PICKS;
-          const version = selectedVersions.get(oracleId);
-          return [
-            version.name,
-            version.image_normal,
-            version.image_flip || null,
-            qualifies && Number.isFinite(rating) && Number.isFinite(picks) ? adjust(picks, rating) : null,
-            Number.isFinite(picks) ? picks : 0,
-            qualifies && Number.isFinite(elo) ? elo : null,
-            Number.isFinite(cubes) ? cubes : 0,
-          ];
-        });
-    }),
-  );
-  const numResultsQ = CardHistory.estimatedDocumentCount(query);
-  const [allData, numResults] = await Promise.all([dataQ, numResultsQ]);
-  const [dataByRating, dataByElo, dataByPicks, dataByCubes] = allData;
+  const dataQ = async () => {
```

Comment:
I don't think this needs to be in an async function. Just move the [data, numResults] to right after CardHistory.find